fedimint-api/src/module/mod.rs

```diff
@@ -440,6 +440,68 @@ where
     }
 }
 
+pub enum ConsensusProposal<CI> {
```

Comment:
Maybe `Force` -> `TriggerNewEpoch` and `Lazy` -> `ContributeToEpoch`

Reply:
I'm not into neither current ones, nor these ones, but the current ones are shorter. :D

Let's get more opinions and I'm fine with whatever.

Reply:
`TriggerEpoch` and `Contribute`?

Reply:
I went with `Trigger` and `Contributed`. Because now we can say that epochs get:

![image](https://user-images.githubusercontent.com/9209/217739366-25ddb8d7-f515-48b4-9a02-4ff222fde23b.png)

Reply:
How about `Force` -> `Trigger` at least?  That is consistent with how we refer to triggering epochs elsewhere in the code.

Reply:
That's what it is now, no? `Trigger` and `Contribute`.